### PR TITLE
fix: Remove redundant querySelector type check

### PR DESCRIPTION
**Pull request by chocolateboy**
Remove a type check which is covered by (and shadows) subsequent checks.

### CODE REVIEW

**Review by btea — APPROVED** [review]

File: packages/happy-dom/src/query-selector/QuerySelector.ts [hunk 1]
```diff
@@ -205,12 +205,6 @@ export default class QuerySelector {
 			);
 		}
```
Comment (capricorn86) [inline packages/happy-dom/src/query-selector/QuerySelector.ts:207]: We should cover the case if selector is a Symbol.

The correct error for Symbol should be:
`TypeError: Failed to execute 'querySelector' on 'Document': Cannot convert a Symbol value to a string`

```suggestion
		if (typeof selector === 'symbol') {
			throw new window.TypeError(
				`Failed to execute 'querySelector' on 'Document': Cannot convert a Symbol value to a string`
			);
		}
```

**Review by capricorn86 — CHANGES_REQUESTED** [review]
Thank you for your contribution @chocolateboy! :star: 

We should cover the case if selector is a Symbol.

File: packages/happy-dom/src/query-selector/QuerySelector.ts (hunk 1 above)
Comment (chocolateboy) [reply packages/happy-dom/src/query-selector/QuerySelector.ts:207]: That check is already included below:

https://github.com/capricorn86/happy-dom/blob/7a5aa6249e7b1fb61aae6ecb64e2ce9384b7e66b/packages/happy-dom/src/query-selector/QuerySelector.ts#L214-L216

Comment (chocolateboy) [reply packages/happy-dom/src/query-selector/QuerySelector.ts:209]: I've updated that error message to include the node type, as per the other checks.

**Review by capricorn86 — APPROVED** [review]
Looks good :slightly_smiling_face:

**Pull request merged**